src/Pillar-PetitPillar.package/PRPillarParserMain.class/class/documentParser.st

```diff
@@ -2,4 +2,4 @@ accessing
 documentParser
 	^ (self useCompiled and: [ Smalltalk globals hasClassNamed: 'PPCompiledParser' ])
 		ifTrue: [ self compiledParser ]
-		ifFalse: [ PRPillarParser ]
\ No newline at end of file
+		ifFalse: [ PRPillarParser new optimize ]
```

Comment:
This is the method that I don't like. PRPillarParserMain should be used as a registration point. And here the usage of the parser is hardcoded. This method should just return the contents of a class variable, and then we should have configuration methods like `usePP2Parser` `usePP1Parser`

Reply:
Can we pair-programm on it?